js/src/routes.js

```diff
@@ -60,6 +62,7 @@ const contractsRoutes = [
 ];
 
 const settingsRoutes = [
+  { path: 'advanced', component: SettingsAdvanced },
```

Comment:
Well it's not really throwing more tabs, since there are no visible links for this section. However, I think that this feature would be quite useful even in production if we add more logging that way. We don't want to pollute the dev console, but having a way to get those logs are sometimes useful (and we don't want to automate this to send them to a log server and whatnot). Nein ?